File: Competitive Programming/DAY-29/metm.py
def time(hh,mm,ss):
    return  hh+(mm/60)+(ss/60)/60
    
def distance():
    m=int(input("Enter the meters"))
    hh=int(input("Enter the hours"))
    mm=int(input("Enter the minutes"))
    ss=int(input("Enter the seconds"))
    miles=(m*0.000621372)/time(hh,mm,ss);
    print("Miles per hour is ",miles)
    
distance()

File: Competitive Programming/DAY-29/metm.cpp
#include <iostream>

using namespace std;

int main()
{
    cout<<"Enter the meters";
    float m;
    cin>>m;
    int hh,mm,ss;
    cout<<"Enter the time taken to cover the distance in HH:MM:SS format";
    cin>>hh>>mm>>ss;
    float hours=hh+(mm/60)+(ss/60)/60;
    float miles=(m*0.000621372)/hours;
    cout<<"Miles per hour is "<<miles;
    return 0;
}
